dist/tools/openocd/openocd.sh

```diff
@@ -57,46 +53,60 @@
 #
 # reset:        triggers a hardware reset of the target board
 #
-#
 # @author       Hauke Peteresen <hauke.petersen@fu-berlin.de>
 # @author       Joakim Nohlgård <joakim.nohlgard@eistec.se>
 
-# default GDB port
-_GDB_PORT=3333
-# default telnet port
-_TELNET_PORT=4444
-# default TCL port
-_TCL_PORT=6333
-# default path to OpenOCD configuration file
-_OPENOCD_CONFIG=${RIOTBOARD}/${BOARD}/dist/openocd.cfg
-# default OpenOCD command
-_OPENOCD=openocd
+# Default GDB port, set to 0 to disable, required != 0 for debug and debug-server targets
+: ${GDB_PORT:=3333}
+# Default telnet port, set to 0 to disable
+: ${TELNET_PORT:=4444}
+# Default TCL port, set to 0 to disable
+: ${TCL_PORT:=6333}
+# Default path to OpenOCD configuration file
+: ${OPENOCD_CONFIG:=${RIOTBOARD}/${BOARD}/dist/openocd.cfg}
```

Comment:
So here it shouldn't point to a generic configuration depending on the debugger?

Reply:
Oops, sorry, I think this change would come in #7686 (still I don't see it there either)

Reply:
In #7686, The config file is the configuration for the board, it should contain the CPU definitions and anything related to getting the board into a state ready to receive a new image. So basically everything except the jtag interface adapter definition, which is handled by the new `$OPENOCD_IFACE_INIT` (in #7686, not here)